### DISCUSSION TITLE
Installation not possible:

### DISCUSSION THREAD

**Discussion by eyoelvici** [nvim-treesitter/nvim-treesitter#7095] [Q&A]
![error](https://github.com/user-attachments/assets/cf1ed5eb-43ea-4d37-b4bc-2957cbef136a) ...vim/lazy/nvim-treesitter/lua/nvim-trees   sitter/install.lua:86: Parser not available for language "xml"
   Installation not possible: ...vim/lazy/nvim-treesitter/lua/nvim-trees 
 sitter/install.lua:86: Parser not available for language "printf"
![error](https://github.com/user-attachments/assets/16e208aa-ef39-4215-9f89-5ac8d7c99718)
![error](https://github.com/user-attachments/assets/5ad27d39-557e-4b0b-a793-282ef2f5d8d9)

**Comment by skiss9** [comment]
If you follow the steps here it might help:
https://github.com/nvim-treesitter/nvim-treesitter/issues/7276

In the config (the part inside the .setup()), you should also add ensure_installed with the languages you want, and incremental_selection and textobjects properties to turn on all features and set your key bindings.

Also run the following commands to diagnose your issue:
:checkhealth nvim-treesitter
:TSConfigInfo

To manually install a parser if it wasn't defined in your config (if you have ensure_installed working properly you shouldn't need to do this):
:TSInstall javascript
